setup.py

```diff
@@ -170,6 +215,24 @@ def get_extensions():
         for mod in ("mpi_xla_bridge", "mpi_xla_bridge_cpu")
     ]
 
+    if sycl_info["compile"] and sycl_info["libdirs"]:
+        extensions.append(
+            Extension(
+                name=f"{CYTHON_SUBMODULE_NAME}.mpi_xla_bridge_xpu",
+                sources=[f"{CYTHON_SUBMODULE_PATH}/mpi_xla_bridge_xpu.pyx"],
+                include_dirs=sycl_info["compile"],
+                library_dirs=sycl_info["libdirs"],
+                libraries=sycl_info["libs"],
+                language="c++",
+                define_macros=[("OMPI_SKIP_MPICXX", "1")],
```

Comment:
What does this do?

Reply:
> What does this do?

- `languages="c++"` Sycl is a C++ API so we have to use C++ compiler to build xpu cython files

- `define_macros=[("OMPI_SKIP_MPICXX", "1")]` this instructs C++ compiler to ignore OpenMPI C++ bindings. This came up during testing of this PR across diffrent MPI implementations. MPI used to have C++ bindings that are now deprecated. However on some linux distros (Fedora 36) OpenMPI comes with enabled C++ bindings. For example on Fedora 36: `ompi_info | grep "C++ binding"` produces: `C++ bindings: yes` . At the beginning we have bindings enabled but there were problems with missing symbols from OpenMPI C++ bindings module and similar problems were encountered by various people on internet . So we decided to switch them off as C++ bindings are deprecated anyway. And this macro `OMPI_SKIP_MPICXX` set to `1` is telling C++ compiler to ignore those bindings.

Reply:
Okay thanks. Could you add that explanation as a comment in the code please?